K2Bridge.Tests.End2End/KustoTestBase.cs

```diff
@@ -118,12 +118,29 @@ public static async Task Init()
             var token = Environment.GetEnvironmentVariable("AAD_TOKEN") ?? GetAADToken(kustoUri);
 
             kusto = new KustoConnectionStringBuilder(kustoUri, kustoDatabase)
-               .WithAadApplicationTokenAuthentication(token);
+                .WithAadApplicationTokenAuthentication(token);
 
             if (!File.Exists("flights.json.gz"))
             {
-                using var wc = new WebClient();
-                wc.DownloadFile("https://raw.githubusercontent.com/elastic/kibana/v6.8.5/src/server/sample_data/data_sets/flights/flights.json.gz", "flights.json.gz");
+                using var hc = new HttpClient();
+                var response =
+                    await hc.GetAsync(new Uri(
+                        "https://raw.githubusercontent.com/elastic/kibana/v6.8.5/src/server/sample_data/data_sets/flights/flights.json.gz"));
+                if (response.IsSuccessStatusCode)
+                {
+                    using (var stream = await response.Content.ReadAsStreamAsync())
+                    {
+                        var fi = new FileInfo("flights.json.gz");
+                        using (var fs = fi.OpenWrite())
```

Comment:
If you're using the "using declaration" above, you can do it here to:

```c#
                    using var stream = await response.Content.ReadAsStreamAsync();
                    var fi = new FileInfo("flights.json.gz");
                    using var fs = fi.OpenWrite();
                    await stream.CopyToAsync(fs);
```
You can upgrade it further by using "await using", which will dispose asynchronously:
```c#
                    await using var stream = await response.Content.ReadAsStreamAsync();
                    var fi = new FileInfo("flights.json.gz");
                    await using var fs = fi.OpenWrite();
                    await stream.CopyToAsync(fs);
```

Also, is that really the minimal amount of code to achieve the same functionallity as WebClient?

Reply:
about your last comment, the answer is yes.